nat-lab/tests/test_lana.py

```diff
@@ -1097,47 +1084,14 @@ async def test_lana_with_vpn_connection(
                     does_not_contain=["0:0:0:0:0:0"],
                     count=1,
                 ),
-                RttValidator(
```

Comment:
Was this removed because it's covered elsewhere?

Reply:
It was added by error, all needed validators are added in `add_rtt_validators`